book_maker/cli.py

```diff
@@ -90,6 +107,13 @@ def main():
         default=False,
         help="allow NavigableStrings to be translated",
     )
+    parser.add_argument(
```

Comment:
maybe add some param check to cofirm --prompt contains  {text} & {language} could be better?